x/stakeibc/keeper/msg_server_liquid_stake.go

```diff
@@ -68,9 +68,9 @@ func (k msgServer) LiquidStake(goCtx context.Context, msg *types.MsgLiquidStake)
 		k.Logger(ctx).Info("failed to send tokens from Account to Module")
 		return nil, sdkerrors.Wrap(sdkerrors.ErrInvalidCoins, "failed to mint stAssets to user")
 	}
-	// create a deposit record of these tokens
-	depositRecord := types.NewDepositRecord(msg.Amount, msg.HostDenom, hostZone.ChainId,
-		sender.String(), types.DepositRecord_RECEIPT)
+	// create a deposit record of these tokens (pending transfer)
+	// TODO: increment amount on deposit record, add creation to end of epoch 
+	depositRecord := types.NewDepositRecord(msg.Amount, msg.HostDenom, hostZone.ChainId, types.DepositRecord_TRANSFER)
```

Comment:
Why a new record? Thought we want to just add new amount to the current record's amount per https://github.com/Stride-Labs/stride/pull/46/commits/54705464e44b6472b915a49d4938e68b25e3a266#diff-e39538e7f27a7bf579cd9b85a103c0f0b86b60b788534295538d0301a9c5dce6R64